atd-etl/app/process/socrata_queries.py

```diff
@@ -70,6 +70,8 @@
             unit_nbr
             crash {
                 crash_date
+                crash_time
+                crash_id
```

Comment:
Added `crash_id` and `crash_time` to demographics datasets

Reply:
👍